src/h3lib/lib/baseCells.c

```diff
@@ -840,6 +842,8 @@ bool _isBaseCellPolarPentagon(int baseCell) {
  * Valid ijk+ lookup coordinates are from (0, 0, 0) to (2, 2, 2).
  */
 int _faceIjkToBaseCell(const FaceIJK* h) {
+    if (!h || h->face < 0 || h->face >= NUM_ICOSA_FACES)
```

Comment:
@isaacbrodsky - Do you think we need guards on `face` here? Or is this value only calculated within the library? If we can't trust `face`, we should probably add guards for `coord.i`, `coord.j`, and `coord.k` as well.

Reply:
I am a little curious what case this is guarding against. While I agree with preventing undefined behavior, I am not sure offhand whether this is a needed check.